src/main/java/de/app/fivegla/integration/fiware/model/StationaryCameraImage.java

```diff
@@ -0,0 +1,67 @@
+package de.app.fivegla.integration.fiware.model;
+
+import de.app.fivegla.integration.fiware.model.api.FiwareEntity;
+import de.app.fivegla.integration.fiware.model.api.Validatable;
+import de.app.fivegla.integration.fiware.model.internal.Attribute;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Represents a Stationary Camera Image.
+ */
+@Slf4j
+public record StationaryCameraImage(
+        String id,
+        String type,
+        Attribute group,
+        Attribute oid,
+        Attribute droneId,
```

Comment:
Umbenennen in `cameraId`, siehe Sidequest. Direkt richtig machen, damit wir nicht später neu anfassen müssen.